providers/MANAGING_PROVIDERS_LIFECYCLE.rst

```diff
@@ -64,63 +62,67 @@ triggers (and the list changes continuously).
 
     GIT apache/airflow/
     └── providers/
-        ├── src/
-        │   └── airflow/
-        │       └── providers/<NEW_PROVIDER>/
-        │           ├── __init__.py
-        │           ├── executors/
-        │           │   ├── __init__.py
-        │           │   └── *.py
-        │           ├── hooks/
-        │           │   ├── __init__.py
-        │           │   └── *.py
-        │           ├── notifications/
-        │           │   ├── __init__.py
-        │           │   └── *.py
-        │           ├── operators/
-        │           │   ├── __init__.py
-        │           │   └── *.py
-        │           ├── transfers/
-        │           │   ├── __init__.py
-        │           │   └── *.py
-        │           └── triggers/
-        │               ├── __init__.py
-        │               └── *.py
-        └── tests/
-            ├── <NEW_PROVIDER>/
-            |   ├── __init__.py
-            |   ├── executors/
-            |   │   ├── __init__.py
-            |   │   └── test_*.py
-            |   ├── hooks/
-            |   │   ├── __init__.py
-            |   │   └── test_*.py
-            |   ├── notifications/
-            |   │   ├── __init__.py
-            |   │   └── test_*.py
-            |   ├── operators/
-            |   │   ├── __init__.py
-            |   │   └── test_*.py
-            |   ├── transfers/
-            |   │   ├── __init__.py
-            |   │   └── test_*.py
-            |   └── triggers/
-            |       ├── __init__.py
-            |       └── test_*.py
-            └── system/<NEW_PROVIDER>/
-                ├── __init__.py
-                └── example_*.py
-
-.. note::
-      The above structure is work in progress and subject to change till Task SDK feature is complete.
+                 ├── pyproject.toml
+                 ├── provider.yaml
```

Comment:
The provider.toml|yaml need to reside _within_ the providers folder

Reply:
Right!